### ISSUE TITLE
[Feature] Add Activation layer

### ISSUE THREAD

**Issue by zaleslaw** [JetBrains/KotlinDL#35]
We need a separate Activation layer because of #34. In many cases, the BatchNorm layer sits between the previous layer with the Linear Activation function and its own non-linear Activation function.

Also, it's important for Transfer Learning on Keras models, due to the widely used separate [Activation ](https://keras.io/api/layers/core_layers/activation/) layer.

**Comment by zaleslaw** [comment]
The Activation Layer + special ReLU layer were added, I close this issue
